static/js/terminal.js

```diff
@@ -178,8 +178,8 @@ function showProcedure() {
         for (let ab of data) {
             let agent = $('#session-id option:selected');
             if (ab.platform === agent.data("platform") && agent.data("executor") === ab.executor) {
-                term.write(atob(ab.test));
-                input = atob(ab.test);
+                term.write(b64DecodeUnicode(ab.test));
+                input = b64DecodeUnicode(ab.test);
```

Comment:
To avoid processing the decoding twice on the same input, can we just simplify via:
```
input = b64DecodeUnicode(ab.test);
term.write(input);
```

Reply:
Makes sense, changed!